velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -245,11 +243,35 @@ std::string toCallSql(
         sql << signature->nameOf(i);
       }
 
-      sql << ") -> " << toCallSql(body, {}, {});
+      sql << ") -> " << toCallSql(body);
     } else {
       VELOX_NYI();
     }
   }
+}
+
+std::string toCallSql(const core::CallTypedExprPtr& call) {
+  std::stringstream sql;
+  sql << call->name() << "(";
+  toCallSqlInternal(call, sql);
```

Comment:
The new function seems to be generate a list of inputs as SQL. It depends only on call.inputs. Let's change the signature to take a list of inputs and rename to something like toCallInputsSql.

Reply:
Fixed